app/views/coronavirus_local_restrictions/show.html.erb

```diff
@@ -49,7 +49,7 @@
   </div>
   <div class="govuk-grid-column-one-third">
     <div class="coronavirus-local-restriction__related-navigation">
-      <%= render 'govuk_publishing_components/components/related_navigation', {
+      <%= render 'govuk_publishing_components/components/contextual_sidebar', {
```

Comment:
Could we tag the related content in content-tagger and pass in the whole content instead, like we do for the breadcrumb?

For example: https://github.com/alphagov/government-frontend/blob/960e79d99c05f48cb110d007b11ad31e7d802970/app/views/shared/_sidebar_navigation.html.erb#L6

Reply:
I did try this and it wouldn't render anything. I wonder if the `parsed_content` thing in your example is doing something special. I'll have a look :)

Reply:
Sorted now that it's been properly tagged in content tagger